plutus-metatheory/changelog.d/20230221_161618_mjaskelioff_abstract_signatures.rst

```diff
@@ -1,32 +0,0 @@
-.. A new scriv changelog fragment.
-..:
-.. Uncomment the header that is right (remove the leading dots).
-..
-.. Removed
-.. -------
-..
-.. - A bullet item for the Removed category.
-..
-# Added
-
-Implemented abstract signatures.
-
-# Changed
-
-Changed type of mkCons builtin to match implementation.
```

Comment:
Intentional?

Reply:
Apparently we also don't have a changelof for `plutus-metatheory`.